app/assets/javascripts/discourse/app/components/d-document.js

```diff
@@ -74,13 +74,12 @@ export default Component.extend({
 
     this._showingLogout = true;
     this.messageBus.stop();
-    bootbox.dialog(
-      I18n.t("logout"),
-      { label: I18n.t("refresh"), callback: logout },
-      {
-        onEscape: () => logout(),
-        backdrop: "static",
-      }
-    );
+
+    this.dialog.alert({
+      message: I18n.t("logout"),
+      confirmLabel: I18n.t("refresh"),
```

Comment:
wouldn't `confirmationLabel` be better? The wording here makes me think it's some kind of action

Reply:
I went with even more verbosity: 

- confirmButtonLabel
- confirmButtonIcon
- cancelButtonLabel